tests/Doctrine/ODM/MongoDB/Tests/Mapping/XmlMappingDriverTest.php

```diff
@@ -17,7 +18,7 @@
 
 class XmlMappingDriverTest extends AbstractMappingDriverTest
 {
-    protected function loadDriver(): XmlDriver
+    protected function loadDriver(): MappingDriver
```

Comment:
I think this change is unnecessary: we're always returning an `XmlDriver`, so the return type was correct.
```suggestion
    protected function loadDriver(): XmlDriver
```

Reply:
@alcaeus Return type narrowing (covariance) was introduced in PHP 7.4 while we are supporting PHP 7.2+. This method is declared as `abstract` in `AbstractMappingDriverTest` with `MappingDriver` return typehint. The type hint was missing before.

Reply:
Oh, I thought we got that in 7.2. Never mind then...